### PR TITLE
Css.media queries

### PR DESCRIPTION
**Pull request by MahtaMirzaei**
Related: #8 
This issue has taken me less than 2 hours so far.

### CODE REVIEW

File: mahta/CSS/CSS responsive design/styles.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
+ @media only screen and (max-width: 600px) and (orientation: landscape) {
+   body{
+      background-color: lightgoldenrodyellow;
```
Comment (mkermani144) [inline mahta/CSS/CSS responsive design/styles.css:3]: I already wondered how do you find the colors you use. Now I also wonder how long a color name can be in CSS.

File: mahta/CSS/CSS responsive design/index.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<head>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+
+ <div class="Title">
```
Comment (mkermani144) [inline mahta/CSS/CSS responsive design/index.html:7]: Please don't use Upperfirst classnames. Stick to either camelCase or better, kebab-case.

File: mahta/CSS/CSS responsive design/styles.css [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      text-align: right;
+      font-size: 1.25rem;
+   }
+ }
+ @media only screen and (min-width: 600px) {
+   body{
+      background-color: lightblue;
+   } 
+    .Title {
+      text-align: center;
```
Comment (mkermani144) [inline mahta/CSS/CSS responsive design/styles.css:32]: By middle of the page, I mean aligning both horizontally and vertically.

**Review by mkermani144 — CHANGES_REQUESTED** [review]

File: mahta/CSS/CSS responsive design/styles.css (hunk 1 above)
Comment (MahtaMirzaei) [reply mahta/CSS/CSS responsive design/styles.css:3]: Vs code's recomendations.. but I think they'll get me bored soon and I'll go back to rgbs😁

Comment (MahtaMirzaei) [reply mahta/CSS/CSS responsive design/styles.css:3]: But there is also this site I've been checking out, it has way longer names for colors. Trust me😂

File: mahta/CSS/CSS responsive design/styles.css (hunk 3 above)
Comment (MahtaMirzaei) [reply mahta/CSS/CSS responsive design/styles.css:52]: I'm not certain if both align-items & justify-content properties were necessary to add, but I wasn't certain which one to use.

**Review by mkermani144 — APPROVED** [review]

**Pull request merged**